scenarios/kubernetes_e2e.py

```diff
@@ -0,0 +1,204 @@
+
+
+def check_output(*cmd):
+    """Log and run the command, return output, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    return subprocess.check_output(cmd)
+
+
+def check(*cmd):
+    """Log and run the command, raising on errors."""
+    print >>sys.stderr, 'Run:', cmd
+    subprocess.check_call(cmd)
+
+
+def sig_handler(_signo, _frame):
+    """Handle signal.SIGTERM and signal.SIGINT."""
+    print 'signo = %s, frame = %s' % (_signo, _frame)
+    check(['docker', 'stop', CONTAINER])
+
+
+def main(args):
+    """Set up env, trigger runner, handle termination. """
+    # pylint: disable=too-many-locals
+
+    lookup = {'workspace':os.environ.get('WORKSPACE')}
+
+    # Get golang into our PATH so we can run e2e.go
+    os.environ['PATH'] += ':/usr/local/go/bin'
+
+    # dockerized-e2e-runner goodies setup
+    workspace = lookup.setdefault('workspace', os.getcwd())
+    artifacts = '%s/_artifacts' % workspace
+    os.makedirs(artifacts)
+
+    e2e_image_tag = 'v20170104-9031f1d'
+    e2e_image_tag_override = '%s/hack/jenkins/.kubekins_e2e_image_tag' % workspace
+    if os.path.isfile(e2e_image_tag_override):
+        with open(e2e_image_tag_override, 'r') as tag:
+            e2e_image_tag = tag.read()
+
+    docker_env_ignore = [
+      'GOOGLE_APPLICATION_CREDENTIALS',
+      'GOROOT',
+      'HOME',
+      'PATH',
+      'PWD',
+      'WORKSPACE'
+    ]
+
+    # exec
+
+    print 'Starting %s...' % CONTAINER
+
+    cmd = ['docker', 'run', '--rm',
+      '--name=%s' % CONTAINER,
+      '-v', '%s/_artifacts":/workspace/_artifacts' % workspace,
+      '-v', '/etc/localtime:/etc/localtime:ro'
+    ]
+
+    # Rules for env var priority here in docker:
+    # -e FOO=a -e FOO=b -> FOO=b
+    # --env-file FOO=a --env-file FOO=b -> FOO=b
+    # -e FOO=a --env-file FOO=b -> FOO=a(!!!!)
+    # --env-file FOO=a -e FOO=b -> FOO=b
+    #
+    # So if you overwrite FOO=c for a local run it will take precedence.
+    #
+
+    if args.env_file:
+        for env in args.env_file:
+            cmd.extend(['--env-file', env])
+
+    gce_ssh = '/workspace/.ssh/google_compute_engine'
+    gce_pub = '%s.pub' % gce_ssh
+    service = '/service-account.json'
+
+    cmd.extend(['-v', '%s:%s:ro' % (args.gce_ssh, gce_ssh),
+                '-v', '%s:%s:ro' % (args.gce_pub, gce_pub),
+                '-v', '%s:%s:ro' % (args.service_account, service),
+                '-e', 'JENKINS_GCE_SSH_PRIVATE_KEY_FILE=%s' % gce_ssh,
+                '-e', 'JENKINS_GCE_SSH_PUBLIC_KEY_FILE=%s' % gce_pub,
+                '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
+
+    # Boilerplate envs
+
+    # Skip gcloud update checking
+    cmd.extend(['-e', 'CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=\'true\''])
+    # Use default component update behavior
+    cmd.extend(['-e', 'CLOUDSDK_EXPERIMENTAL_FAST_COMPONENT_UPDATE=\'false\''])
+
+    cmd.extend(['-e', 'E2E_UP=%s' % args.e2e_up,
```

Comment:
nit: if you're using multiple lines please do a line break beforehand so that we do not need to adjust things if you change the name of cmd to commmand.

Aka if you do this:
```
change_cmd.extend(['if I change this line',
               'Note how this line is now misaligned'])

change_cmd.extend([
    'However with this strategy',
    'the padding does not change when I change the first line'])
```